### PR TITLE
Add admin_user setting in host file

### LINKED ISSUE #203: Set admin_users configuration option with ansible hosts configuration file
Some users need access to the `Admin` menu to manage end-user servers.

Add instructions and configuration options to allow users to add users to the standard `JupyterHub.admin_users` group.

### PR DESCRIPTION
**Pull request by jgwerner**
Closes #203

### CODE REVIEW

File: ansible/hosts.example [hunk 1]
```diff
@@ -59,6 +59,12 @@ all:
       #   lti11_enabled and lti13_enabled are mutually exclusive, therefore
       #   set one to true to enable the desired lti auth standard.
 
+      ### Genaral Options
+      #-------------------
+      # provide a list of admin users separated by commas. Usernames should represent
+      # the normalized username with logging into the system.
+      # admin_users: foo,biz,bar
```
Comment (netoisc) [inline ansible/hosts.example:66]: does it accept single or multiple values? in other files i see the `admin_user` name

File: ansible/group_vars/all.yml [hunk 2]
```diff
@@ -56,6 +56,9 @@ docker_proxy_image: "{{ docker_proxy_image_param | default('jupyterhub/configura
 ## Jupyter Notebook config
 jupyter_notebook_config: "{{ jupyter_notebook_config_param | default('jupyter_notebook_config.py') }}"
 
+# Admin users
+admin_user: "{{ admin_user_param | default('') }}"
```
Comment (netoisc) [inline ansible/group_vars/all.yml:60]: If this is correct we have to change the hosts.example to use the variable as plural

File: ansible/hosts.example (hunk 1 above)
Comment (jgwerner) [reply ansible/hosts.example:66]: You're right. My initial intention was to support multiple values but for this first version, its simpler (and safer) to only accept one value.

File: ansible/group_vars/all.yml (hunk 2 above)
Comment (jgwerner) [reply ansible/group_vars/all.yml:60]: I changed the item mentioned above, so will leave this setting as-is.

File: ansible/hosts.example (hunk 1 above)
Comment (netoisc) [inline ansible/hosts.example:66]: should it be plural ? in `ansible/roles/jupyterhub/templates/env.jhub.j2` file is used as singular

Comment (jgwerner) [reply ansible/hosts.example:66]: It should be single, yes. I'll provide an update.

**Comment by jgwerner** [comment]
Will re open with a fresh branch.

**Pull request closed without merging**